generation/spade_gan/spade_gan.ipynb

```diff
@@ -0,0 +1,2665 @@
+{
```

Comment:
<p>The level of the title of this cell can be the same as that of the previous cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>